packages/react/src/components/Loader/Loader.tsx

```diff
@@ -105,14 +105,22 @@ class Loader extends UIComponent<ReactProps<LoaderProps>, LoaderState> {
     const { indicator, label } = this.props
     const { visible } = this.state
 
+    const svgElement =
+      styles.svg && Box.create({}, { defaultProps: { className: 'container', styles: styles.svg } })
```

Comment:
I don't like the fact the we are changing the structure of the element depending of something that exists in the styles. The user should be aware trough the API of the component that they can specify svg element... More over the same can be created trough the indicator slot anyways. Can you suggest why do you see these changes necessary?